### DISCUSSION TITLE
Ngx editor Image upload

### DISCUSSION THREAD

**Discussion by Padmmaja** [sibiraj-s/ngx-editor#332] [Q&A]
Hi, I am displaying images which i collect from electron in ngxeditor as img src = imagepath+ image using ngfor loop
Now in ts file how to get the html content including img src tag that i can update this hrml content in my json.
Please help! Stuck with this issue

**Comment by sibiraj-s** [comment]
I will help here. but can you rephrase your question. I am not sure what you wanted to achieve here.

**Comment by Padmmaja** [comment]
Hi Sibiraj,

Thank you,

I am using ngx editor. I am uploading multiple images, i want to see the
output if this editor as htrm content.

Like this: <p>this is text<p><img src=c:/users/images/tulips.jpg>

But i get output as like this now:  <p>this is
text<p><c:/users/images/tulips.jpg></p>

Please kindly help.

Let me know if i can share code screenshots

Regards,
Padmaja

**Reply by sibiraj-s** [reply]
Got it. 

The built in menu only has option to insert images of given URL. How do you upload your images to the editor now?

**Reply by sibiraj-s** [reply]
@Padmmaja are you still facing the issue? If not. I can mark the discussion as complete.

**Comment by Padmmaja** [comment]
Very inhuman of you

**Comment by Padmmaja** [comment]
Very irresponsible and very inhuman, i should not have mailed to you and
wasted my time, you got no decency to reply after requesting so many times.  Sir if you remember, you mailed saying you will help if i explain my issue, after that you did not replied to any of mail, forget even acknowledging 

Cant believe educated ppl be like this, because of ppl like you, many
taking it granted that they dont have to reply and can waste others  time.

Certainly you will repay someday

**Reply by sibiraj-s** [reply]
Please be patient. I think you haven't seen my reply. I replied as soon as you did. see here https://github.com/sibiraj-s/ngx-editor/discussions/332#discussioncomment-546338. I waited for few hours for your reply before I left. thinking that you might have figured it out.

**Reply by Padmmaja** [reply]
Sibiraj, dont know what you trying to tell, check my screenshot of the email, pls show where is your reply for this email?
![Uploading  #6B4E47ED-C47C-4480-8E5B-C4F635BD7AD8.png…]()

![2CDCAEA5-B6F0-42D7-A251-AAF8FE68FC1B](https://user-images.githubusercontent.com/81588114/113977809-c091e000-9860-11eb-9614-28d8ae8f61a5.png)

**Reply by sibiraj-s** [reply]
I understand that you didn't get the email notification for my reply, may be you can check your notification settings. But I did reply to you. You can open this thread on a browser https://github.com/sibiraj-s/ngx-editor/discussions/332 to check. I feel sorry that I couldn't help you that day.

**Reply by sibiraj-s** [reply]
>  pls show where is your reply for this email

Sorry. It is not me who sends those mail. It is Github that does that. The overall conversation happens via Github discussions.

[a comment by an automated account is not shown]
